src/dune_rules/melange/melange_rules.ml

```diff
@@ -172,6 +172,18 @@ let build_js ~loc ~dir ~pkg_name ~mode ~module_systems ~output ~obj_dir ~sctx
            ; Dep src
            ]))
 
+(** attach [deps] to the specified [alias] AND the (dune default) [all] alias.
```

Comment:
```suggestion
(* attach [deps] to the specified [alias] AND the (dune default) [all] alias.
```

Reply:
reflected in 7c49c1d7027915cc9aff8d3a76fa58081ba84ecd